quail/processes/wps_climdex_quantile.py

```diff
@@ -100,6 +99,20 @@ def collect_args_wrapper(self, request):
 
         return [data_file] + literal_inputs
 
+    def unpack_data_file(self, data_file, data_vector):
+        try:
+            return load_rdata_to_python(data_file, data_vector)
+        except (RRuntimeError, ProcessError, IndexError):
+            pass
+
+        try:
+            return robjects.r(f"unlist(readRDS('{data_file}'))")
+        except (RRuntimeError, ProcessError) as e:
+            raise ProcessError(
+                f"{type(e).__name__}: Data file must be a RDS file or "
+                "a Rdata file containing an object of the given name"
+            )
```

Comment:
This looks good though nice work! What exactly does `unlist` do?

Reply:
I believe it simplifies 'a list like structure' according to the [docs](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/unlist). Sangwon used it for the vector name when passing an RDA to the process:
```Python
with NamedTemporaryFile(suffix=".rda", prefix="quantile_", dir="/tmp", delete=True) as output_file:
    output = quail.climdex_quantile(
            data_file=resource_filename("tests","data/ec.1018935.rda"),
            data_vector="unlist(ec.1018935.tmax['MAX_TEMP'])",
            quantiles_vector="c(0.1, 0.5, 0.9)",
            vector_name="tmax_quantiles",
        )
quantile_url = output.get()[1]
``` 
But RDS doesn't take a vector name so it can't be done that way.